lang/README.md

```diff
@@ -1 +1,16 @@
-A directory for language-specific parsers.
+# Language Specific Parsers
+
+This directory contains a set of language-specific parsers, that can be used to generate a class and file level dependency graph of your project. It outputs this dependency graph as a protobuff. For users of bfg, this dependency graph can be used to generate your BUILD files.
```

Comment:
"that can be used to generate a class..." -> that are used to provide BFG with dependency information.

"protobuff" - we can link to the definition, https://github.com/bazelbuild/BUILD_file_generator/blob/672c5572499e96f6a89bfaa5d7baaf92184c6d7c/src/main/java/com/google/devtools/build/bfg/bfg.proto#L9

"bfg" -> "BFG" ?